man/man8/zpool.8

```diff
@@ -2044,6 +2062,17 @@ Because scrubbing and resilvering are \fBI/O\fR-intensive operations, \fBZFS\fR
 .RS 6n
 Stop scrubbing.
 .RE
+.sp
+.ne 2
+.na
+\fB\fB-p\fR\fR
+.ad
+.RS 6n
+Pause scrubbing. Scrub progress is periodically synced to disk so if the system
```

Comment:
That's true, maybe it isn't necessary to note anything here.  Its just those of us who know that internally scrub and resilver are nearly the same, we might wonder but we can also just check the code :-)